web/client-api/src/main/java/io/deephaven/web/client/api/CoreClient.java

```diff
@@ -0,0 +1,85 @@
+package io.deephaven.web.client.api;
+
+import elemental2.promise.Promise;
+import io.deephaven.web.client.api.storage.JsStorageService;
+import jsinterop.annotations.JsType;
+
+@JsType(namespace = "dh")
+public class CoreClient extends QueryConnectable<CoreClient> {
```

Comment:
Most of this file is not ready for prime time, but exists to grant access to `getStorageService()`. I'd be happy to remove or comment out most of it if preferred, or just expect changes in the near future with auth/config tickets.